README.md

```diff
@@ -1 +1,51 @@
-# Review_Project_Trial
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Review_Project_Trial
+
+> One paragraph statement about the project.
+
+![screenshot](./admission.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html and CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
Why not remove this placeholder text as well or replace it with content so your README is more professional?

Reply:
thanks for the code review I'll make the changes you suggested